lib/query-builders/applicant.js

```diff
@@ -6,19 +6,33 @@ const {
   withdrawnByApplicant
 } = require('../flow/status');
 
+const isApplicant = profile => builder => builder
```

Comment:
This hurts my eyes. Can we wrap it in some curlies?

Reply:
#jode

Reply:
hahaha

Reply:
FINE.